server/utils/logger.ts

```diff
@@ -0,0 +1,12 @@
+import pino from "pino";
+
+let environment = (process.env.NODE_ENV) || "development";
+
+// Parent logger. If you want to make use of logging in a new file, you MUST import this and create a child logger
+// See index.ts, db.ts, or scraper.ts for examples
+const logger = pino({
+    level: process.env.LOG_LEVEL || "info",
```

Comment:
Just wondering but since the other log levels are set as a env variable, does this mean we can't use other log levels directly in code? E.g. `log.log()` or `log.debug()`

Reply:
oh yeah good question! the env variable is actually just there to set the minimum log level that we can see and it defaults to `"info"`

you can see the levels [here](https://github.com/pinojs/pino/blob/master/docs/api.md#levels), but basically, setting it to `"info"` means that you can see all log levels except for `"trace"` and `"debug"`

choosing info was just an arbitrary choice on my part LOL. we can change the default to something else though.

Reply:
Oh that makes sense, thanks! `info` sounds good w/ me